src/BlazorUI/Bit.BlazorUI/Components/Inputs/ColorPicker/BitColorPicker.scss

```
@@ -257,7 +257,7 @@
 
     &:focus-visible {
         @include focus-ring;
-        border-radius: spacing(0.25);
+        border-radius: $shp-radius-control;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)